### ISSUE TITLE
Missing license?

### ISSUE THREAD

**Issue by fullhamalti** [Gamefreak130/CommonUtils#1]
Please add a license.

See https://choosealicense.com/no-permission/

**Comment by Gamefreak130** [comment]
Consider: these are helper functions for modding a game that is nearly 13 years old, written in a language that is trivial to decompile from a published assembly. As twallan once said, "No such thing as trade secrets, in a system where anyone can read your code-base."

No one is going to sue you for reuse or modification, least of all myself. In and of itself, there is almost zero value to this code, and thus no market you could usurp from me, so what would be the point? In fact, I _encourage_ its reuse; I would be _honored_ if a commercial project used my code for whatever reason. All I ask is that you be a good sport and credit me for the help.

TL;DR: Live a little, embrace the legal grey area :)
